code/game/objects/items/weapons/storage/wallets.dm

```diff
@@ -66,12 +80,12 @@
 
 
 /obj/item/weapon/storage/wallet/GetID()
-	return front_id
+	return front_id // net_id is only a holder for accesses.

```

Comment:
all the comments on your code are useless, they just make the code harder to read. Remove them.